src/main/java/org/embulk/parser/jsonpath/JsonpathParserPlugin.java

```diff
@@ -187,4 +194,22 @@ private void skipOrThrow(DataException cause, boolean stopOnInvalidRecord)
         }
         logger.warn(String.format(ENGLISH, "Skipped invalid record (%s)", cause));
     }
+
+    // this method is to keep the backward compatibility of 'schema' option.
+    private SchemaConfig getSchemaConfig(PluginTask task)
+    {
+        if (task.getOldSchemaConfig().isPresent()) {
+            logger.warn("Please use 'columns' option instead of 'schema' because the 'schema' option is deprecated. The next version will stop 'schema' option support.");
```

Comment:
Thanks. I merged it into `else if (task.getOldSchemaConfig().isPresent())` part.